guides/common/modules/con_planning-project-upgrade.adoc

```diff
@@ -0,0 +1,24 @@
+[id="planning-{project-context}-upgrade"]
+= Planning {Project} upgrade
+
+Upgrading to {Project} {ProjectVersion} affects your entire {Project} infrastructure.
+Plan carefully before proceeding.
+
+* Read the {ProjectName} {ProjectVersion} {ReleaseNotesDocURL}[Release Notes].
+* Consider whether any of your scripts need updating.
```

Comment:
```suggestion
* Consider whether any of your integrations need updating.
```

IMHO this better encapsulates the fact that it can be bash scripts, ansible playbooks or some other api-based stuff.